services/scan/src/workers/interpret_vx.ts

```diff
@@ -55,6 +59,7 @@ export async function configure(store: Store): Promise<void> {
   debug('creating a new interpreter');
   interpreter = new Interpreter({
     electionDefinition,
+    precinctSelection: ALL_PRECINCTS_SELECTION,
```

Comment:
Is there a reason why we're using `ALL_PRECINCTS_SELECTION` here instead of `store.getPrecinctSelection()`?

Reply:
This is a really helpful catch @arsalansufi, I think this was a mistake.

Reply:
Ah nice!

Reply:
Fixed.

Reply:
@adghayes were the tests passing with this mistake? If so, that might be a bit concerning